packages/actor-init-query/lib/cli/CliArgsHandlerBase.ts

```diff
@@ -56,6 +56,12 @@ export class CliArgsHandlerBase implements ICliArgsHandler {
       sourceString = sourceString.slice(0, authMatches.index + 2) +
         sourceString.slice(authMatches.index + credentials.length + 3);
     }
+
+    if (sourceString.startsWith('+')) {
```

Comment:
This microsyntax is not that self-explanatory, and may cause confusion in the future. So I would recommend adding a dedicated CLI argument for this.

Reply:
Indeed, however this parameter should be used for specific uris. Any idea on how to map this?

Reply:
We probably need a more expressive CLI approach to define source-level context options.
Is this something that you need for your use case?
If not, I would suggest leaving this open for future work (and making an issue(, and just enabling this option for _all_ sources for now.